src/verilog/verilog_interfaces.cpp

```diff
@@ -163,14 +163,14 @@ void verilog_typecheckt::interface_inst(
   const verilog_instt::instancet &op)
 {
   bool primitive=statement.id()==ID_inst_builtin;
-  const exprt &range=static_cast<const exprt &>(op.find(ID_range));
+  const exprt &range_expr = static_cast<const exprt &>(op.find(ID_range));
 
-  mp_integer msb, lsb;
-  
-  if(range.is_nil() || range.id().empty())
-    msb=lsb=0;
+  ranget range;
+
+  if(range_expr.is_nil() || range_expr.id().empty())
+    range.msb = range.lsb = 0;
```

Comment:
Shouldn't there be a default constructor of `ranget` that does this?

Reply:
Considered it, but would you remember whether msb is the first argument, or lsb?

Reply:
I'll go for msb, lsb.